### PR TITLE
Feature/profile page

### PR DESCRIPTION
**Pull request by toc2menow**
image

@goohooh 께서 router 붙여주시고 제가 user status를 가져와서 현재 렌탈중인 책을 보여주는 부분을 작업했습니다.

### CODE REVIEW

File: frontend/src/components/ProfilePage/ProfilePage.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  align-items: center;
+`;
+
+const ProfileContent = styled.div`
+ width: 100%;
+  max-width: 720px;
+`;
+
+const NowReading = styled.div`
+  font-wiehgt: bold;
```
Comment (Goblinlordx) [inline frontend/src/components/ProfilePage/ProfilePage.tsx]: :cat:

Comment (Goblinlordx) [reply frontend/src/components/ProfilePage/ProfilePage.tsx]: D= meant to wait for this to be fixed to approve T-T sorry (itchy trigger finger)

**Comment by Goblinlordx** [comment]
Just noticed failing frontend tests.  Unrelated to this PR I think but it allows merge with failures?? @thefron

**Comment by thefron** [comment]
@Goblinlordx updated the setting

**Comment by goohooh** [comment]
@thefron @Goblinlordx Test failed because of what we've installed

**Comment by Goblinlordx** [comment]
I thought I fixed all the `withTheme` stuff.  That needs to be changed for updated version of MUI.

**Comment by Goblinlordx** [comment]
image
These kinds of changes need to be reverted.

File: frontend/src/components/ProfilePage/ProfilePage.tsx (hunk 1 above)
Comment (Goblinlordx) [reply frontend/src/components/ProfilePage/ProfilePage.tsx]: @toc2menow `font-weight` ^^

**Pull request merged**